composer.json

```diff
@@ -25,8 +25,9 @@
     "require-dev":  {
         "pimple/pimple": "~1.0",
         "symfony/phpunit-bridge": "~2.7",
-        "silex/silex":  "~1.0",
-        "twig/twig":    "~1.16|~2.0"
+        "symfony/routing": "~2.3",
```

Comment:
please allow 3.0 here too (and for the bridge), so that it would be tested against them